### PR TITLE
JBPM-9273 System error occurred while adding deployment unit

### PR DESCRIPTION
**Pull request by bxf12315**
**Thank you for submitting this pull request**

**JIRA**: [JBPM-9273](https://issues.redhat.com/browse/JBPM-9273)

**Referenced Pull Requests**:
* https://github.com/kiegroup/droolsjbpm-integration/pull/2235
* link 2
* link 3 etc.

**Business Central**: [WAR file](https://donwnload.here/something)

**VS Code**: [plugin](https://donwnload.here/something)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</details>

### CODE REVIEW

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
@nmirasch @prashuts 
PR:https://github.com/kiegroup/kie-wb-common/pull/3417, has different branch from https://github.com/kiegroup/droolsjbpm-integration/pull/2235,let FDB do no has two of them. So raise a new PR to fix it.

**Comment by prashuts** [comment]
@bxf12315  PR https://github.com/kiegroup/kie-wb-common/pull/3417 is still valid to review ? or can we discard ?

**Comment by bxf12315** [comment]
> @bxf12315 PR #3417 is still valid to review ? or can we discard ?

I will close it.

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by bxf12315** [comment]
jenkins retest this

**Review by prashuts — APPROVED** [review]
Looks good to me

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/test/java/org/kie/workbench/common/screens/server/management/client/container/ContainerPresenterTest.java [hunk 1]
```diff
@@ -197,6 +197,7 @@ public void testInit() {
     public void testStartContainer() {
         presenter.loadContainers(containerSpecData);
 
+
```
Comment (nmirasch) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/test/java/org/kie/workbench/common/screens/server/management/client/container/ContainerPresenterTest.java:200]: @bxf12315 no needed line?

**Review by nmirasch — APPROVED** [review]
feel free to fix the super minor comment. Looks good to me

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/test/java/org/kie/workbench/common/screens/server/management/client/container/ContainerPresenterTest.java (hunk 1 above)
Comment (bxf12315) [reply kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/test/java/org/kie/workbench/common/screens/server/management/client/container/ContainerPresenterTest.java:200]: Done

**Comment by bxf12315** [comment]
@prashuts Can you check it?

**Comment by bxf12315** [comment]
jenkins do fdb

**Comment by prashuts** [comment]
jenkins do fdb

**Pull request merged**